client/state/data-layer/test/wpcom-api-middleware.js

```diff
@@ -7,14 +7,20 @@ import { spy, stub } from 'sinon';
 /**
  * Internal dependencies
  */
-import { middleware } from '../wpcom-api-middleware';
 import { local, mergeHandlers } from '../utils';
+import useMockery from 'test/helpers/use-mockery';
 
 describe( 'WordPress.com API Middleware', () => {
+	let middleware;
 	let next;
 	let store;
 
+	useMockery( ( mockery ) => {
+		mockery.registerMock( './analytics', {} );
+	} );
```

Comment:
why are we mocking this here? it seems like a bad indication if we have to do this…